src/components/campaign-types/Categories.ts

```diff
@@ -0,0 +1,13 @@
+export enum CampaignTypeCategory {
```

Comment:
This file better be lowercase as we don't export any React component from it, but it acts as utility function

Minor detail